tests/nestedfor.c

```diff
@@ -0,0 +1,11 @@
+#include<stdio.h>
```

Comment:
Please follow the consistent coding style as file `tests/struct.c`.
File `tests/for.c` is inconsistent, and you can substitute the original with your `nestedfor.c`.